packages/sql/src/aggregates.ts

```diff
@@ -0,0 +1,161 @@
+import { SQLExpression, parseSQL, sql } from "./expression";
```

Comment:
Is that any way you can mark the files as moved? Maybe rename first and commit and then change the content. It would make it easier to see the differences.

Reply:
I agree, I'm a little time-strapped for the next few days but will see if I can do some sort of rebase + fix commit log to make it easier to compare the diff.

Reply:
Maybe the easiest is to make a new branch, do the rename, commit, paste in the new files, commit again. Replace this branch.

However, now that I think about it, I wonder whether renames will be tracked correctly when we squash.

Reply:
I'm kind of surprised that GitHub doesn't show the files as renamed. Are there so many changes?

Reply:
@domoritz I have a cleaned up version here: https://github.com/uwdata/mosaic/compare/main...calvinfo:calvinfo/sql/ts-2?expand=1. Lmk what you think. I'm happy to make a PR to that if you think it looks good.

Reply:
Thanks. It still looks like some of the files are not tracked as renamed like packages/sql/src/datetime.js.

Reply:
Yeah, not sure what to tell you there. I separated it out this time into two explicit rename commits (no functionality change)

https://github.com/uwdata/mosaic/commit/37a2b9259caa06bd44ee673785e675d5ef75b65e
https://github.com/uwdata/mosaic/commit/bc4d19ce05ce281f15a8e4c7c01f4b0de19581a8

In the case of `datetime.js`, I think a high enough percentage of the lines changed to trigger a "re-write" vs a "rename" in the github UI.

Reply:
Thanks for separating the changes. Once this works, let's merge two commits. One with all the renames, and one with the code changes. That should be good!